yt/frontends/gadget/testing.py

```diff
@@ -96,7 +96,8 @@ def fake_gadget_binary(
             # Generate and write field block
             data = []
             for pt in ptype:
-                data += [np.random.rand(npart[pt], dim)]
+                rng = np.random.default_rng()
+                data += [rng.random((npart[pt], dim))]
```

Comment:
Any reason to not move `rng` outside the loop?

i.e., 
```
            rng = np.random.default_rng()
            for pt in ptype:                
                data += [rng.random((npart[pt], dim))]
```

Reply:
Done !